lib/renderer/web-view/web-view.js

```diff
@@ -42,6 +42,11 @@ var WebViewImpl = (function () {
       this.webviewNode.setZoomLevel(zoomLevel)
     }
     webFrame.on('zoom-level-changed', this.onZoomLevelChanged)
+
+    this.onVisibilityChanged = (event, visibilityState) => {
+      this.webviewNode.send('ELECTRON_RENDERER_WINDOW_VISIBILITY_CHANGE', visibilityState)
+    }
+    ipcRenderer.on('ELECTRON_RENDERER_WINDOW_VISIBILITY_CHANGE', this.onVisibilityChanged)
```

Comment:
We can probably listen to the DOM `visibilitychange` event, to reduce code duplication.